src/ol_concourse/pipelines/open_edx/mfe/pipeline.py

```diff
@@ -84,6 +85,7 @@ def mfe_params(
             f"{open_edx.environment}-open-edx-language-preference"
         ),
         "LMS_BASE_URL": f"https://{open_edx.lms_domain}",
+        "LEARNING_BASE_URL": f"https://{open_edx.learning_base_url}",
```

Comment:
I think we can make this change more robust like. This way we would not need to add this to `src/ol_concourse/pipelines/open_edx/mfe/values.py`. So, the only change would be this one line.

```suggestion
        "LEARNING_BASE_URL": f"https://{open_edx.learning_base_url}/{mfe.application.path}",
```
OR

```suggestion
        "LEARNING_BASE_URL": f"https://{open_edx.learning_base_url}/learn",
```

Reply:
Yeah, this 2nd suggestion seems best. I am updating it.

Reply:
I am not sure about a way to get the learning MFE path dynamically here. @mitodl/devops Could suggest one maybe?

Reply:
I just pushed a commit that will automatically pull in the proper value. I started to write out how to do it, but it was easier to just show you with code :)

Reply:
Thanks, This makes much more sense now.